ninetofiver/choices.py

```diff
@@ -0,0 +1,38 @@
+from ninetofiver.settings import REDMINE_URL, REDMINE_API_KEY
+from redminelib import Redmine
+from requests.exceptions import ConnectionError
+
+
+def get_redmine_project_choices():
+    if REDMINE_URL is not '' and REDMINE_API_KEY is not '': 
```

Comment:
You are allowing a 'None' value, but not an empty string? It's also better to import settings and do `settings.REDMINE_URL`, instead of depending on your settings being set at interpreter initialization.